nototools/noto_cmap_reqs.py

```diff
@@ -2042,6 +2065,21 @@ def _generate_script_extra(script_to_chars):
   # Orya - Oriya
 
   # Osge - Osge (Osage)
+  ('Osge',
+   # Comment
+   """
+   Added by Monotype.
+   """,
+   # Data
+   """
+   # Combining Diacritical Marks
+   0301  # COMBINING ACUTE ACCENT
+   0304  # COMBINING MACRON
+   030B  # COMBINING DOUBLE ACUTE ACCENT
+   0358  # COMBINING DOT ABOVE RIGHT
+   # Geometric Shapes
+   25CC  # DOTTED CIRCLE
```

Comment:
Yeah, I'm going to add that but decided to keep it as a separate cl.  These changes originate with Monotype, any 25CC changes would originate with us.